src/test/deploy/functions/release/fabricator.spec.ts

```diff
@@ -481,6 +488,97 @@ describe("Fabricator", () => {
       );
     });
 
+    it.only("handles already existing eventarc channels", async () => {
```

Comment:
What does `it.only` do rather than `it`?

Reply:
Ah I forgot to take that out, hadn't cleaned up the PR before creating the draft!

If some unit tests are specified with `it.only`, only they will run when the `npm run test` command is executed.